src/data/vse_structural_data_cleaning_script.py

```diff
@@ -0,0 +1,36 @@
+train['OverallQual'].replace([1, 2, 3, 4], '4-', inplace=True)
```

Comment:
I prefer to create a function and done all these changes in the dataframe inside it.

Reply:
Didn't understand what you mean. You mean creating a function that substitutes the replace method?

> Good job on the data cleaning. My principal feedback is for you add some comments about what you are doing. Put commens like "how it was", "what Im gonna do to change it". In this way other people can understand better.

Ok! I'm gonna try to do this more often and more clearly next time.

> > The commits you made were much better that last time! Continue improving your git skills hahaha!

Thanks!